### PR TITLE
fix(connectors): fixes MGDCTRS-733: fixed reconcile interval configuration for connector integration tests in all environments

### PR DESCRIPTION
**Pull request by dhirajsb**
## Description
Connector integration test harness explicitly sets reconcile interval to 1 second, and not just rely on the config when using a mock OCM server. 

## Verification Steps
CI tests should pass, nightly tests should pass after PR is merged. 

## Checklist (Definition of Done)
- [X] All acceptance criteria specified in JIRA have been completed
~~- [ ] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)~~
~~- [ ] Documentation added for the feature~~
- [x] CI and all relevant tests are passing
- [x] Code Review completed
~~- [ ] Verified independently by reviewer~~
~~- [ ] Required metrics/dashboards/alerts have been added (or PR created).~~
~~- [ ] Required Standard Operating Procedure (SOP) is added.~~~~
~~- [ ] JIRA has created for changes required on the client side~~

### CODE REVIEW

**Review by machi1990 — APPROVED** [review]

**Comment by machi1990** [comment]
The PR lgtm. 
I've approved it. You might need an approval from someone from the connector team as well. 
It'll be good to get this in soon-ish to see if nightly CI will be fixed - it should be but we can only find out until when this is merged.

**Review by lburgazzoli — APPROVED** [review]

**Comment by lburgazzoli** [comment]
@machi1990 done

**Pull request merged**
